importer/importer_utils.py

```diff
@@ -434,3 +434,16 @@ def create_wlm_url(country, language, id):
     url_base = "https://tools.wmflabs.org/heritage/api/api.php?action=search&format=json&srcountry={}&srlanguage={}&srid={}"
     return url_base.format(
         country, language, quote(id))
+
+
+def get_P31(q_number):
+    results = []
+    site = pywikibot.Site("wikidata", "wikidata")
+    repo = site.data_repository()
+    item = pywikibot.ItemPage(repo, q_number)
+    item_dict = item.get()
```

Comment:
Replace this and the next line with with just `item.exists()` then use `if item.claims.get("P31")`